src/core/thread/mesh_forwarder.cpp

```diff
@@ -1814,11 +1814,14 @@ void MeshForwarder::LogMessage(MessageAction       aAction,
     switch (aAction)
     {
     case kMessageReceive:
-    case kMessageTransmit:
     case kMessagePrepareIndirect:
         logLevel = (aError == kErrorNone) ? OT_LOG_LEVEL_INFO : OT_LOG_LEVEL_NOTE;
         break;
-
+    case kMessageTransmit:
+        logLevel = (aError == kErrorNone || aError == kErrorNoAck || aError == kErrorChannelAccessFailure)
+                       ? OT_LOG_LEVEL_INFO
+                       : OT_LOG_LEVEL_NOTE;
```

Comment:
I feel like not a good model to be selective about certain errors and use different log level per error. I think it would be cleaner to use same log level for all error types. 

The error here in `MeshForwarder` is after all MAC retries (when we are really dropping the message). So they should not be as common as `Mac` frame tx errors. So it may be fine to keep them at `NOTE` level.
If we think there may be too many at `NOTE` then I suggest we do all errors at `INFO`.

Reply:
Agree. 
Reverted to `NOTE` for now. Will continue to monitor this log.